assembly/massa/datastore_manipulations/main.ts

```diff
@@ -7,28 +7,44 @@
  * which is ready to be send on Massa network node!
  **/
 
-import { Storage, getKeys, getKeysOf, generateEvent, Context, print } from "@massalabs/massa-as-sdk";
+import {Storage, getKeys, getKeysOf, generateEvent, Context, print} from "@massalabs/massa-as-sdk";
 
 export function main(_args: string): void {
```

Comment:
I would recommend writing a second test and keep this one as simple as possible :)

Reply:
Separated the test in two halfs with a comment in between, tell me if that's OK for you